### ISSUE TITLE
Serving Hash (#) URIs

### ISSUE THREAD

**Issue by white-gecko** [white-gecko/jekyll-rdf#5]
DTP16/jekyll-rdf#41

If a URI contains a hash (`#`) this is usually not resolved by the server, but by the browser.
Consider following URIs:

```
http://example.org/page
http://example.org/page#somethingelse
http://example.org/page#something
http://example.org/other-page#header1
http://example.org/other-page#header2
```

Result:
In this case two pages should be generated:

```
http://example.org/page
http://example.org/other-page
```

while the template should care about listing the hash resources on this page.

**Comment by white-gecko** [comment]
This is somehow related to #9

**Comment by white-gecko** [comment]
Fixed in #39 resp. #42
